src/2_RTB/VerbaliInterni/VerbaleInterno_211123/content.typ

```diff
@@ -0,0 +1,41 @@
+Il team si impegna a mantenere la Proponente costantemente informata sull'avanzamento durante lo Sprint, con un aggiornamento settimanale. Tale aggiornamento avverrà idealmente a metà Sprint; tuttavia, in caso di urgenze o necessità particolari, il team contatterà la Proponente per ottenere il loro _feedback_ in momenti aggiuntivi. Inoltre, comprendendo l'interesse dei referenti della Proponente nell'analizzare il codice sviluppato prima della _Sprint Retrospective_, il team si impegna a contattarli anche poco prima della fine dello Sprint, immediatamente dopo aver completato il _merge_ del _feature branch_ di interesse nel _branch_ di derivazione nella _repository_ InnovaCity.
+
+== Numerosita' dei Verificatori
+Il team ha valutato l'assegnazione del ruolo di Verificatore, decidendo di limitarlo a un solo membro per Sprint successivi al primo. Questa scelta mira a garantire una maggiore enfasi e supporto ai ruoli le cui mansioni sono attualmente più pressanti, come il Programmatore o l'Analista.
+Tuttavia, nel caso si decida in futuro di coinvolgere più di un Verificatore, la politica per assegnare loro le _issue_ da revisionare all'interno dell'ITS è la seguente: il Verificatore seleziona autonomamente la _issue_ con la priorità maggiore all'interno della corsia _Ready to Review_ sulla Kanban Board in utilizzo su Github e procede con la revisione.
+
+== Pianificare date di inizio e fine per ogni _issue_
+Al termine di ogni meeting interno la prassi è quella di creare e assegnare le _issue_ all'interno dell'ITS, consentendo la definizione di parametri come la dimensione (_Size_) e la priorità (_Priority_) di ciascuna. In data odierna si è deciso di concordare anche la data di inizio e soprattutto di fine di ogni _issue_, al fine di pianificare l'avanzamento del lavoro in maniera più dettagliata, considerando scadenze più immediate rispetto alla fine dello Sprint corrente. Questa pratica mira a facilitare il monitoraggio delle interconnessioni tra le diverse _issue_ e favorire una collaborazione più efficace tra i membri del team, permettendo una pianificazione più precisa e agevolando la gestione delle dipendenze tra le varie attività.
+
+== Lavorare in team di due persone
+Il team è intenzionato ad adottare un approccio per cui determinate _issue_ vengono assegnate a coppie di persone all'interno del team in un futuro prossimo; tuttavia, nello Sprint corrente le responsabilità dei componenti sono prevalentemente distinte, con un'eccezione per i due Analisti. In altre parole, l'associazione tra membri avviene solo nel caso in cui due individui si trovino a ricoprire lo stesso ruolo. Quando l'implementazione del codice richiederà un ruolo più attivo da parte dei Programmatori e coinvolgerà almeno due persone, queste lavoreranno in coppia. In sostanza, l'approccio collaborativo di due persone per task specifiche verrà maggiormente adottato quando le attività richiederanno un coinvolgimento più diretto dei Programmatori.
```

Comment:
Il lavoro in coppia non e' per forza in pair-programming sulla stessa issue, anzi speriamo che solo poche volte sia cosi', altrimenti si ha un grosso spreco di tempo. Il lavoro in coppia penso sia inteso piu' come, fisicamente o digitalmente ci si trova per lavorare e magari chiedere consigli al volo, non per fare un lavoro in due.
Almeno io avevo inteso questo

Reply:
Comprensibile, avevo inteso male io pensando che avremmo assegnato issue di una certa portata a coppie di persone in modo che potessero lavorarci assieme ma effettivamente e' piu' sensato suddividere in issue piu' piccole e al massimo spronare la collaborazione. Ho semplificato e modificato la sezione per riflettere la tua osservazione.